torch/testing/_internal/common_methods_invocations.py

```diff
@@ -64,7 +64,8 @@ def __init__(self,
                  dtypesIfROCM=None,  # dtypes this function is expected to work with on ROCM
                  test_inplace_grad=True,  # whether to gradcheck and gradgradcheck the inplace variant
                  skips=tuple(),  # information about which tests to skip
-                 decorators=None):  # decorators to apply to generated tests
+                 decorators=None,  # decorators to apply to generated tests
+                 supports_out_param=True):  # does it support out=... ?
```

Comment:
Let's put this just under `test_inplace_grad` and above `skips` and `decorators`. 

Comment would be "whether the op supports the out kwarg" to be parallel with `test_inplace_grad` comment.

Reply:
fixed by 8eac798787